platform/admin/admin-configuration-configinstaller/src/main/java/org/codice/ddf/admin/configuration/FelixConfig.java

```diff
@@ -69,29 +86,46 @@ File getFelixFile() {
     }
 
     /**
-     * Sets the felix file name using the default technique from the config repo, using .config in
-     * case complex data structures are present.
-     *
-     * @throws IOException if an error occurs persisting to config admin.
-     */
-    void setFelixFile() throws IOException {
-        setFelixFile(createFelixFileName(config));
-    }
-
-    /**
      * Sets the felix file name to the provided {@link File} argument.
      *
      * @param file the file to use for the felix file name property.
-     * @throws IOException if an error occurs persisting to config admin.
      */
-    void setFelixFile(File file) throws IOException {
+    void setFelixFile(File file) {
         Dictionary<String, Object> propsWithFelixFileName = config.getProperties();
         propsWithFelixFileName.put(FELIX_FILENAME_PROP,
```

Comment:
@brjeter Good point! It also identifies a problem I commented above about making sure we only process one event at a time.